tests/kernel/mutex/sys_mutex/src/main.c

```diff
@@ -429,7 +431,11 @@ void test_main(void)
 	k_mem_domain_add_thread(&ztest_mem_domain, THREAD_09);
 	k_mem_domain_add_thread(&ztest_mem_domain, THREAD_11);
 #endif
-	sys_mutex_lock(&not_my_mutex, K_NO_WAIT);
+	rv = sys_mutex_lock(&not_my_mutex, K_NO_WAIT);
+	if (rv != 0) {
+		TC_ERROR("Failed to take mutex %p\n", &not_my_mutex);
```

Comment:
Is it intended that this will not fail the testcase?
If this ever fails, sanitycheck will not detect this as a failure..
i.e. hacking the condition to `if(1) {` you'd get
```
***** Booting Zephyr OS build zephyr-v1.14.0-3283-g62fa472919a7 *****
FAIL - test_main@436. Failed to take mutex 0x5660f1b4
===================================================================
PROJECT EXECUTION SUCCESSFUL
```

Reply:
@aescolar Thanks. I will change it.